projects/js-packages/components/components/record-meter-bar/index.tsx

```diff
@@ -106,6 +107,30 @@ const RecordMeterBar: React.FC< RecordMeterBarProps > = ( {
 					} ) }
 				</ul>
 			</div>
+			<table
+				className="screen-reader-text"
+				summary="This is the screen reader alternative data table for the record meter bar graphic."
+			>
+				<caption>{ __( 'Records for Record Meter Bar', 'jetpack' ) }</caption>
+				<tbody>
+					<tr>
```

Comment:
I think we probably want to adjust the table markup we're outputting here. I don't think each of the post types should be a `th`.

Perhaps something more like:

```
<tr>
  <th scope="col">Record type</th>
  <th scope="col">Record count</th>
</tr>
<tr>
  <td>record type one</td>
  <td>123</td>
</tr>

.... and then a new `<tr>` for each record type.

Reply:
Okay sure. I was on the fence about this one myself, but ended up rolling with an example I found elsewhere using the label as a heading. But making the post types <td> does make more sense in this context 👍

Reply:
👍 Either seems reasonable. If we did want to use the post types as a row heading, I reckon the markup should be more like:

```
<tr>
  <th scope="col">Record type</th>
  <th scope="col">Record count</th>
</tr>
<tr>
  <th scope="row">record type one</th>
  <td>123</td>
</tr>
```

"Record type" and "Record count" have a column scope because the header applies to the column beneath them. 

"record type one" has a row scope because the header applies to the row it's on.